src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1719,6 +1719,52 @@ paths:
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
 
+  /state/account-locker/page/account-resources:
+    post:
+      operationId: TBD-state-account-locker-page-account-resources
+      summary: TBD
+      description: TBD
+      tags:
+        - State
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/StateAccountLockerPageAccountResourcesRequest"
+      responses:
+        "200":
+          description: TBD
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StateAccountLockerPageAccountResourcesResponse"
+        "4XX":
+          $ref: "#/components/responses/ErrorResponse"
+
+  /wallet/tbd: # TODO ok, we need to figure out suitable naming scheme, and then rename some types
+    post:
+      operationId: TBD-wallet-tbd-tbd
+      summary: TBD
+      description: TBD
+      tags:
+        - State
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/WalletTbdRequest"
+      responses:
+        "200":
+          description: Validators Uptime
```

Comment:
nitpick: TBD :D

Reply:
Descriptions and actual endpoint paths are beyond the scope of the schema proposal.